app/models.py

```diff
@@ -20,14 +21,155 @@
 from .env import load_env
 from .json_util import to_json
 
-session_context_id = contextvars.ContextVar("request_id", default=None)
-DBSession = scoped_session(sessionmaker(), scopefunc=session_context_id.get)
-
 env, cfg = load_env()
 strict = cfg["security.strict"]
 use_webhook = cfg["security.slack.enabled"]
 webhook_url = cfg["security.slack.url"]
 
+session_context_id = contextvars.ContextVar("request_id", default=None)
+# left here for backward compatibility:
+DBSession = scoped_session(sessionmaker(), scopefunc=session_context_id.get)
+
+
+class Session(sa.orm.session.Session):
+    """
+    Create an instance of Session when you
+    want to apply a verification method on all added
+    or modified or deleted rows before commiting them
+    to the database.
+
+    This class overrides the commit() function
+    by adding a verify() function before it.
+
+    Use this in a context manager:
+    with VerifiedSession(user_object) as session:
```

Comment:
Are you still planning to rename to VerifiedSession as indicated here?

Reply:
It is renamed in the function call below the class definition.